server/odc-service/src/main/java/com/oceanbase/odc/service/flow/task/model/DatabaseChangeParameters.java

```diff
@@ -47,6 +47,8 @@ public class DatabaseChangeParameters implements Serializable, TaskParameters {
     @NotNull
     private Boolean generateRollbackPlan;
     private boolean modifyTimeoutIfTimeConsumingSqlExists = true;
+    // internal usage for notification
+    private String parentTaskType;
```

Comment:
TaskType will be better

Reply:
The parent task may be a SqlPlanJob , it's a `JobType` instead of `TaskType`. So this should use String.

Reply:
done